src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -405,6 +405,16 @@ public function compileEchoDefaults($value)
     }
 
     /**
+     * Replace the @parent directive to a placeholder.
+     *
+     * @return string
+     */
+    protected function compileParent()
```

Comment:
It's called from here: https://github.com/laravel/framework/blob/5.3/src/Illuminate/View/Compilers/BladeCompiler.php#L327